Helper/Elasticsearch/ClientRequestRuntime.php

```diff
@@ -77,4 +80,37 @@ public function data(string $input)
 
         return ($total > 1) ? false : null;
     }
+
+    /**
+     * @param string $input
+     *
+     * @return array|false|null false when multiple results
```

Comment:
too much copy paste here, false is never returned. 
Remove the docblock completely and use the return type hinting instead. That's never inconsistent.

Reply:
See the commit 68c481f3d1b4f029a73b6f277e90c7b577204fed is it better like this?